tardis/montecarlo/estimators/estimator.py

```diff
@@ -0,0 +1,49 @@
+from tardis.montecarlo.estimators.util import bound_free_estimator_array2frame
+import tardis.constants as const
+
+H = const.h.cgs.value
+
+
+class ContinuumEstimators:
```

Comment:
`EstimatedContinuumProperties`

Reply:
`ContinuumMCEstimators`

Reply:
Ah - In the end we decided for `ContinuumEstimators` as this is used by both MC and analytical methods